raiden/tests/integration/rpc/assumptions/test_rpc_gas_price_assumptions.py

```diff
@@ -155,24 +222,67 @@ def test_remote_transaction_with_zero_gasprice_is_not_mined(
         raise RuntimeError(f"Unknown blockchain_type {blockchain_type}")
 
 
-def test_duplicated_transaction_different_gas_price_raises(deploy_client: JSONRPCClient) -> None:
+def test_resending_pending_transaction_with_lower_gas_raises(deploy_client: JSONRPCClient) -> None:
```

Comment:
I just renamed the test to make it clear what it is doing and changed the code to match the style of the other tests.